OpenRA.Mods.Common/Traits/TooltipDescription.cs

```diff
@@ -41,7 +49,7 @@ public bool IsTooltipVisible(Player forPlayer)
 			if (IsTraitDisabled)
 				return false;
 
-			var stance = forPlayer.Stances[self.Owner];
+			var stance = Owner == null || forPlayer == null ? Stance.None : Owner.Stances[forPlayer];
```

Comment:
You coulda try hardcoding this to Stance.Allies instead of Stance.None for now and adding a comment with a HACK prefix above it to mention it being a workaround. It might hide the neutral/enemy tooltips - I haven't tested if that would work.

Reply:
If that doesn't work, then just hide them all for spectators IMO.